.gitlab-ci.yml

```diff
@@ -165,11 +165,11 @@ variables:
   # To use images from datadog-agent-buildimages dev branches, set the corresponding
   # SUFFIX variable to _test_only
   DATADOG_AGENT_BUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_BUILDIMAGES: v26473479-2dfb49af
+  DATADOG_AGENT_BUILDIMAGES: v26894465-ddbd51cf
   DATADOG_AGENT_WINBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_WINBUILDIMAGES: v26473479-2dfb49af
+  DATADOG_AGENT_WINBUILDIMAGES: v26894465-ddbd51cf
   DATADOG_AGENT_ARMBUILDIMAGES_SUFFIX: ""
-  DATADOG_AGENT_ARMBUILDIMAGES: v26473479-2dfb49af
```

Comment:
Do we have a rule or requirement to bump all images at the same time? I can't think of a scenario where partial updates this would break anything, but I've always seen all images being updated at once

Reply:
Maybe for testing purpose on a PR? If I update something in the windows build image and just push this one to a registry?

But in general I think we always bump them at once yup